### PR TITLE
use xattrs when installing apk package and when creating tarball

### PR DESCRIPTION
**Pull request by deitch**
As described. And tests, of course.

### CODE REVIEW

**Comment by deitch** [comment]
The one thing I am a little unsure of is the value of the xattr in the tar header. The tar header has PAXRecords as `map[string]string`, while the xattrs on disk are `map[string][]byte` (or the native equivalent).

I have been treating the value as just `string([]byte)`, but that might not be correct. There might be some kind of conversion involved.

**Review by imjasonh — APPROVED** [review]
This seems like a good candidate for an e2e test, e.g., that a package that has capabilities, packaged by apko, has those capabilities in the final image.

I think this is fine to add like this, but wdyt about a more expansive test?

File: pkg/apk/install.go [hunk 1]
```diff
@@ -209,6 +220,18 @@ func (a *APK) installAPKFiles(gzipIn io.Reader, origin, replaces string) ([]tar.
 		default:
 			return nil, fmt.Errorf("unsupported file type %s %v", header.Name, header.Typeflag)
 		}
+
+		// xattrs
+		for k, v := range header.PAXRecords {
+			if !strings.HasPrefix(k, "SCHILY.xattr.") {
+				continue
+			}
+			attrName := strings.TrimPrefix(k, "SCHILY.xattr.")
```
Comment (mattmoor) [inline pkg/apk/install.go:229]: Should this be using the constant defined above?

Comment (deitch) [reply pkg/apk/install.go:229]: Yeah, it should. Separate PR coming

**Pull request merged**